### DISCUSSION TITLE
Wrapped Bitcoin (WBTC)

### DISCUSSION THREAD

**Discussion by luziusmeisser** [Frankencoin-ZCHF/Frankencoin#17] [Acceptable Collaterals]
We should support Wrapped Bitcoin as a collateral.

What are the right parameters for a position with a three year horizon?

CHF interest [stands at 1.75%](https://de.statista.com/statistik/daten/studie/1124625/umfrage/leitzins-der-schweizer-nationalbank/#:~:text=Leitzins%20der%20Schweizer%20Nationalbank%20bis%20Dezember%202023&text=Am%2021.,Am%2013.), and the yield curve for three year mortgages [stands at 2.02%](https://www.hypotheke.ch/zinskurve-hypotheken-zinsen/#zinskurve,current:-1), so let's start with 2%.

On top of that, we need to add a risk premium that depends on the volatility of the underlying asset, the reserve ratio, and the auction duration. These three define the expected outcome in case the market price P touches the liquidation price LIQ in the following chart:

![image](https://github.com/Frankencoin-ZCHF/FrankenCoin/assets/3128389/55d777d0-f78b-428a-b281-3be770756bdb)

Let's assume the auction duration is two days, with each of the two phases being 1 day. This gives the position owner (and the market) one day to react in case the position is being challenged. This should be enough given the liquidity of WBTC even if there are only very few system participants. Currently, Bitcoin's annualized volatility [stands at 56%](https://www.theblock.co/data/crypto-markets/prices/annualized-btc-volatility-30d). This translates into a two-day volatility of 0.56*sqrt(2/365)=4.1% or an hourly volatiliy of 0.6%. Given these parameters, we are interested in the expected outcome of successful challenges. So the question is: given that the price touched the liquidation price and a challenge was started, what is the expected profit or loss for the system?

Plugging the above numbers into a Monte Carlo simulation, it looks like we can expect successful auctions to and at a price of around 96% of the liquidation price. This means that already with a buffer of 10%, the expected liquidation will be slightly profitable. To be on the safe side (considering that returns are not normally distributed in practice), let's set the buffer at 20%, yielding an average profit of 14% of the liquidation price (after deducting 2% challenger reward). This means that we do not need to add a risk premium at all at average with a buffer as generous as 20%. Even when simulating with historical data, we find that a 10% buffer is enough to make liquidations profitable most of the time:

![image](https://github.com/Frankencoin-ZCHF/FrankenCoin/assets/3128389/234bb5c1-c61b-4643-aa0c-9ec001d37869)
(Chart from the Frankencoin thesis.)

With a buffer of 20%, the minter gets 80%, so when setting the interest to 2%, the effective interest is 2%/80%=2.5%, which should be a reasonably attractive rate. What do you think?

**Comment by TaprootFreak** [comment]
I am in favour of adding WBTC.

I find it difficult to have an opinion on the question of conditions. I will try to add some data. Perhaps it will be helpful.

On the credit side:
I currently have an interest rate of 0% on my Raiffeisen account. Apparently we have got used to the fact that the bank doesn't pay us any interest on overnight deposits. With my USDT in my Metamask, I have also got used to not receiving any interest on my USD holdings.

On the debt side:
I have a mortgage with the following parameters:
max 80% loan-to-value
term of 3 years
Saron base rate 1.692064%
Raiffeisen surcharge 0.75%
Effective interest rate: 2.442064%

Bitcoin has a slightly higher volatility than the housing market. So if I get an offer from Frankencoin for 80% loan-to-value and 2.5% interest, then I think that's very attractive.

**Comment by torginmackinga** [comment]
WBTC seems like a good collateral since it is very liquid. 

The limit should not be set too high (compared to the total supply of ZCHF), as there is a tail risk that the custodian fails and the coins become unbacked.
